### ISSUE TITLE
Warning message not accessible by screenreader

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#241907]
Testing #241777

On Windows with NVDA, I am able to tab back to the attached mp4 with Shift+Tab, but I don't see any indication of the attachment being invalid.
